32-bit pre-setup fails

We're just starting to see the 32-bit pre-setup fail, e.g.:
https://github.com/ocaml-multicore/multicoretests/actions/runs/4977631855/jobs/8906911507
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libc6:i386 : Depends: libgcc-s1:i386 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Error: Process completed with exit code 100.
```

I wonder if some Ubuntu package had it's dependencies changed over the last few day :thinking:

--- a/.github/workflows/common.yml
+++ b/.github/workflows/common.yml
@@ -132,7 +132,7 @@ jobs:
           if [ "${COMPILER##*32bit}" = "" ]; then
             sudo dpkg --add-architecture i386
             sudo apt-get update
-            sudo apt-get install pkg-config:i386 libzstd1:i386 libzstd-dev:i386
+            sudo apt-get install pkg-config:i386 libzstd1:i386 libzstd-dev:i386 libgcc-s1:i386
           fi
 
           # Generate an OPAM config for a custom compiler

